manifests/01_namespace.yaml

```diff
@@ -13,5 +13,7 @@ metadata:
     pod-security.kubernetes.io/enforce: baseline
     pod-security.kubernetes.io/enforce-version: v1.25
     pod-security.kubernetes.io/audit: restricted
+    pod-security.kubernetes.io/audit-version: v1.25
```

Comment:
One question, as you know, the master branch aligns with OCP 4.13 which is based on K8s 1.26, so any reason to use `1.25` here? Or do we plan to update it to `1.26` after OCP 4.13 rebase to K8s 1.26? Thanks!

Reply:
The plan is to set it to `v1.25` (to match `enforce-version`), back port to 4.12, then update to `v1.26` at the proper time.

Reply:
Got it, thanks!